_dashboards/visualizations.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Visualizations
+nav_order: 40
+---
+
+# Visualize your data with dashboards
+
+OpenSearch Dashboards gives you the opportunity to explore and interpret information in an interactive visual environment. Having the right visual can help you accurately show metrics and data patterns in a clear, unbiased way.  
+
+The following highlights the OpenSearch Dashboards visualizations types and typical usage.
+
+-   Graphs and charts
```

Comment:
Could we get some PM help on the actual OpenSearch use cases for all of these? Like (from our docs):
Gantt charts are useful in trace analytics, telemetry, and anomaly detection use cases, where you want to understand interactions and dependencies between various events in a schedule.